.rubocop.yml

```diff
@@ -1,6 +1,8 @@
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.4 # lowest supported version
+  NewCops: disable
```

Comment:
prefer enable and disable the ons that cause trouble explicitly (or leave a TODO for that)